contracts/governance/TimelockController.sol

```diff
@@ -434,4 +453,19 @@ contract TimelockController is AccessControl, ERC721Holder, ERC1155Holder {
         emit MinDelayChange(_minDelay, newDelay);
         _minDelay = newDelay;
     }
+
+    /**
+     * @dev Encodes a `OperationState` into a `bytes32` representation where each bit enabled corresponds to
+     * the underlying position in the `ProposalState` enum. For example:
+     *
+     * 0x000...1000
+     *   ^^^^^^----- ...
+     *         ^---- Done
+     *          ^--- Ready
+     *           ^-- Waiting
+     *            ^- Unset
+     */
+    function _encodeStateBitmap(OperationState proposalState) internal pure returns (bytes32) {
+        return bytes32(1 << uint8(proposalState));
```

Comment:
For consistency, we need to call the arguments with the contract-related concepts. In this case it's an operation although it's a proposal when it passes through governance.

```suggestion
    function _encodeStateBitmap(OperationState operationState) internal pure returns (bytes32) {
        return bytes32(1 << uint8(operationState));
```